deploy/helm/sumologic/values.yaml

```diff
@@ -608,7 +608,8 @@ fluent-bit:
     #   cpu: 10m
     #   memory: 8Mi
 
-  enabled: true
+  ## Set the enabled flag to false to not install fluent-bit helm chart as a dependency along with this helm chart
```

Comment:
```suggestion
## Uncomment the flag below to not install fluent-bit helm chart as a dependency along with this helm chart.
## Comment it out if you wish to have the fluent-bit dependency installed.
## Setting the flag to true instead of commenting out will break other functionality. 
```